Documentation/check-compat-table.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
```

Comment:
Renaming this to `check-crd-compat-table.sh` might make it's function more clear. Not a big issue, though, since the script is documented elsewhere.